app/views/curation_concerns/base/_chf_show_page.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="work-show-page-thumb">
```

Comment:
i believe the answer is that chf_show_page isn't actually used anymore and should be removed! good catch.